### PR TITLE
fix: Some new visual bugs caused by recent changes

### PR DESCRIPTION
**Pull request by jsimplicio**
Before:
screen shot 2016-03-15 at 6 33 15 pm
After:
screen shot 2016-03-15 at 6 32 26 pm

### CODE REVIEW

File: public/assets/app.css [hunk 1]
```diff
@@ -363,7 +363,7 @@ table a {
 }
 
 table tr td:first-child {
-  width: 100px;
+  width: 150px;
```
Comment (gr2m) [inline public/assets/app.css]: why the 150px? Wouldn’t it make sense to give it the same width as the Amount input?

**Comment by gr2m** [comment]
just the minor comment on the amount cell width in the table, but otherwise LGTM :)

File: public/assets/app.css (hunk 1 above)
Comment (jsimplicio) [inline public/assets/app.css]: With the font increase to `18px` when you're inside edit the number gets cut off at this width.

Comment (gr2m) [inline public/assets/app.css]: gotcha, good point, then let’s leave it that way

**Comment by nickcolley** [comment]
LGTM :+1:

**Comment by nickcolley** [comment]
The approval system thinks updating the branch requires more LGTM hmm

**Pull request merged**